app/models/wizard/steps/authenticate.rb

```diff
@@ -26,6 +28,12 @@ def timed_one_time_password=(value)
         super(value)
       end
 
+      def candidate_identity_data
+        @store.fetch(IDENTITY_ATTRS).transform_keys do |k|
```

Comment:
Would it make more sense to go via the `@wizard` here to avoid recreating the logic to map to API attribute keys? something like:

```
IDENTITY_ATTRS = %i[email firstName lastName dateOfBbirth].freeze

def candidate_identity_data
  @wizard.export_camelized_hash.fetch(IDENTITY_ATTRS)
end
```

Reply:
This was my first inclination - and actually to put it in the api support mixin but the resend action doesn't have a wizard step setup?

Reply:
Dependancy on wizard causes other issues so leaving as is